cmd/helm/helm.go

```diff
@@ -74,12 +71,32 @@ func main() {
 	}
 }
 
-func bootstrap(c *cobra.Command, args []string) {
+func setupConnection(c *cobra.Command, args []string) error {
+	if tillerHost == "" {
+		// Should failure fall back to default host?
```

Comment:
I think the current implementation is good. Falling back to the default host might be a confusing thing for the user.